### PR TITLE
feat: add signup button and register activity

### PR DESCRIPTION
**Pull request by ClaudiuBogdan**
Add signup button and register activity

### CODE REVIEW

File: app/src/main/res/values-ro-rRO/strings.xml [hunk 1]
```diff
@@ -19,4 +19,10 @@
     <!-- Errors -->
     <string name="error_generic">A apărut o problemă!</string>
 
+    <!-- Login screen -->
+<!--    Tu ți-ai făcut planul în caz de urgență?-->
+    <string name="button_signup" translatable="false">Inscrie-te acum</string>
+    <string name="button_login" translatable="false">Autentifica-te</string>
+    <string name="title_login" translatable="false">Tu ți-ai făcut planul în caz de urgență?</string>
+    <string name="subtitle_login" translatable="false">Hai sa il facem impreuna!</string>
```
Comment (aniri) [inline app/src/main/res/values-ro-rRO/strings.xml]:
```suggestion
    <string name="button_signup" translatable="false">Înscrie-te acum</string>
    <string name="button_login" translatable="false">Autentifică-te</string>
    <string name="title_login" translatable="false">Tu ți-ai făcut planul în caz de urgență?</string>
    <string name="subtitle_login" translatable="false">Hai să îl facem împreună!</string>
```

File: app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt [hunk 2]
```diff
@@ -33,7 +33,7 @@ class SplashScreenActivity : BaseAnalyticsActivity<SplashScreenViewModel>() {
             progressDialog.dismiss()
             val activity = when {
                 loginStatus.isLoggedIn && loginStatus.onboardingCompleted -> MainActivity::class.java
-                loginStatus.isLoggedIn -> OnboardingActivity::class.java
+                loginStatus.isLoggedIn -> RegisterActivity::class.java
```
Comment (aniri) [inline app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt:36]: Why was this change made? if the user is currently logged in, they should be redirected to the onboarding related screens.

Comment (ClaudiuBogdan) [reply app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt:36]: Sorry, de error was due to a rename  from OnboardingActivity to RegisterActivity while creating RegisterActivity

**Comment by aniri** [comment]
@ClaudiuBogdan could you please remove the top action bar from the login and register activities?

image

File: app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt (hunk 2 above)
Comment (aniri) [reply app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt:36]: hmm this still shows as RegisterActivity :/

Comment (ClaudiuBogdan) [reply app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt:36]: fixed

**Review by aniri — APPROVED** [review]

**Pull request merged**